### PR TITLE
Show default and possible options in CLI reference documentation

### LINKED ISSUE #5692: Show possible options in CLI reference documentation
If an option accepts values we should display them, as in `uv help`.

### PR DESCRIPTION
**Pull request by eth3lbert**
## Summary

Closes #5692 .

### CODE REVIEW

**Comment by eth3lbert** [comment]
Partial screenshot:

image

**Comment by eth3lbert** [comment]
Currently the `cli.md` is a bit hard to read without a browser. 😂

File: crates/uv-dev/src/generate_cli_reference.rs [hunk 1]
```diff
@@ -240,6 +242,49 @@ fn generate_command<'a>(output: &mut String, command: &'a Command, parents: &mut
     parents.pop();
 }
 
+fn default_option(opt: &clap::Arg, output: &mut String) {
```
Comment (zanieb) [inline crates/uv-dev/src/generate_cli_reference.rs:245]: Can you prefix these with `emit` (i.e., `emit_default_option`) for clarity?

**Comment by zanieb** [comment]
Did you base this format on someone elses? Trying to figure out what the best way to display it is.

**Comment by eth3lbert** [comment]
> Did you base this format on someone elses? Trying to figure out what the best way to display it is.

This is based on the output of `uv help python`.

**Comment by zanieb** [comment]
That's good enough for me! We can revisit latter if we want, i.e. this is different from our settings reference page.

**Review by zanieb — APPROVED** [review]

**Comment by zanieb** [comment]
I kind of prefer it as-is, I think. We'll see.

**Pull request merged**